db/migration/1676022628012-AddPostsWithGdocPublishStatusView.ts

```diff
@@ -0,0 +1,20 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class AddPostsWithGdocPublishStatusView1676022628012
+    implements MigrationInterface
+{
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`-- sql
```

Comment:
ooc, what's up with the `-- sql` comments ?

Reply:
With [this extension](https://marketplace.visualstudio.com/items?itemName=qufiwefefwoyn.inline-sql-syntax) you get SQL syntax highlighting if you prefix strings in programming languages with `-- sql`